tests/performance/src/test/java/org/mule/Base64EncodingPerformanceTestCase.java

```diff
@@ -0,0 +1,55 @@
+package org.mule;
+
+import static org.mule.util.Base64.encodeBytes;
+
+import org.mule.tck.junit4.AbstractMuleContextTestCase;
+import org.mule.util.Base64;
+
+import java.io.IOException;
+
+import org.databene.contiperf.PerfTest;
+import org.databene.contiperf.Required;
+import org.databene.contiperf.junit.ContiPerfRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class Base64EncodingPerformanceTestCase extends AbstractMuleContextTestCase
```

Comment:
why is this necessary? It doesn't look like your fix would affect performance that much. The problem with this type of tests is that given our "heavy loaded" CI environment it's very likely to get flacky